server/routers/work-shares/generateWalletActivationChallenge.ts

```diff
@@ -19,6 +19,29 @@ export const generateWalletActivationChallenge = protectedProcedure
     // operation will probably reuse it. Otherwise, the cleanup cronjobs will take care of it:
     const deviceAndLocationIdPromise = getDeviceAndLocationId(ctx);
 
+    // Explicitly set JWT claims before wallet lookup
+    try {
+      // Set JWT claims with session-level SET
+      const jwtClaims = JSON.stringify({
```

Comment:
Based on the latest comments, this is already included in `ensureJwtClaimsSet`, right?